decentraland/unity-renderer
# refactor: biw plugin folder move

## What does this PR change?

This PR moves the builder in world plugin to the DCLPlugins

The folder looks like that
![image](https://user-images.githubusercontent.com/54064092/142395321-baf37d27-e4e5-4096-995c-59f37d11e475.png)


## How to test the changes?


1. Go to: https://play.decentraland.zone/?renderer-branch=refactor/biw-plugin-move
2. Check that builder in world is working as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
